lib/database_connection.rb

```diff
@@ -0,0 +1,11 @@
+require 'pg'
+
+class DatabaseConnection
+  def self.setup(dbname)
+    @connection = PG.connect(dbname: dbname)
+  end 
+
```

Comment:
This can condense to methods into one.
Ensure that you have two DB's
- chitter_development
- chitter_test

```
def self.exec(sql_query)
    begin
      connection = PG.connect(dbname: "chitter_#{ENV['RACK_ENV']}")
      connection.exec(sql_query)
    rescue PG::Error => e
      puts e.message
    ensure
      connection.close if connection
    end
  end
```